velox/exec/Task.cpp

```diff
@@ -1411,7 +1411,28 @@ void Task::addOperatorStats(OperatorStats& stats) {
   taskStats_.pipelineStats[stats.pipelineId]
       .operatorStats[stats.operatorId]
       .add(stats);
-  stats.clear();
+}
+
+TaskStats Task::taskStats() const {
+  std::lock_guard<std::mutex> l(mutex_);
+
+  // This will contain task stats plust stats for the completed operators.
+  TaskStats taskStats = taskStats_;
+
+  // Add stats of the operators that are still running.
```

Comment:
Can you add a test to verify we are now collecting stats from a running query. you can use testValue to block a query'e execution for test purpose. Thanks!